tasks/image_configuration/downgrade-wpa-hostapd/manifest.yaml

```diff
@@ -0,0 +1,11 @@
+config: DS_DOWNGRADE_WPA_HOSTAPD
+tasks:
+  - cmd: fetch.sh
+    cmd_type: host
+    description: Fetching binary packages
+  - cmd: install.sh
+    cmd_type: target
+    dependencies:
+      - DS_CHROOT_EARLY
+      - DS_APT_CONFIGURED
```

Comment:
Each cmd_type has a default dependency.  Eg, can't run on the target before the docker is ready, so it had to have some sane defaults:
https://github.com/embeddedTS/distro-seed/blob/main/common/build.py#L64

CHROOT_READY is from here:
https://github.com/embeddedTS/distro-seed/blob/main/tasks/core/chroot_ready/manifest.yaml

Which just makes sure apt-get install -f has completed. I think you could omit these dependencies entirely.  It would install the old tools, then run this to downgrade them.